mw-webapp/src/converter/querySnapshot.ts

```diff
@@ -0,0 +1,6 @@
+import {DocumentData, QuerySnapshot} from "firebase/firestore";
+
+/**
+ * QuerySnapshot type
```

Comment:
looks like model, not converter

Reply:
it's not model. it's type of querySnapshot I move out it to separate file because I use it in each converter file. I will delete it if I will create a generic for querySnapshot

Reply:
it's a model

Reply:
Moved